client/components/ScopeDropdown/ScopeDropdown.tsx

```diff
@@ -178,7 +179,9 @@ const ScopeDropdown = (props: Props) => {
 												mode: 'settings',
 											})}
 										>
-											<Icon icon={pubPubIcons.settings} iconSize={12} />
+											<Tooltip content="View settings">
```

Comment:
I think it would be equally clear to drop "View" here (and elsewhere) and just say "Settings".